silta/silta-next.yml

```diff
@@ -18,6 +18,7 @@ services:
       DRUPAL_REVALIDATE_SECRET: ${DRUPAL_REVALIDATE_SECRET}
       AUTH_SECRET: ${AUTH_SECRET}
       NEXT_PUBLIC_DRUPAL_BASE_URL: https://<|DRUPAL_DOMAIN|>
+      DRUPAL_BASE_URL_INTERNAL: https://<|DRUPAL_DOMAIN|>
```

Comment:
@Jancis how do we get here the internal url for drupal?

Reply:
it should be `<release-name>-drupal`

Reply:
is there a risk for the release name to be too long because of the branch name and this failing for that?

Reply:
so we should add another task after this one: 

https://github.com/wunderio/next-drupal-starterkit/blob/0066792accc71ea8ca79f32dcc47314b8fb7e19a/.circleci/config.yml#L307 


Where we again replace the value of the env var DRUPAL_BASE_URL_INTERNAL to this new value...